MAINTAINERS.md

```diff
@@ -1,17 +1,19 @@
 # OpenSearch Anomaly Detection Maintainers
 
 ## Maintainers
-| Maintainer | GitHub ID | Affiliation |
-| --------------- | --------- | ----------- |
-| Hanguang Zhang | [zhanghg08](https://github.com/zhanghg08) | Amazon |
-| Jack Mazanec | [jmazanec15](https://github.com/jmazanec15) | Amazon |
-| Jing Zhang | [jngz-es](https://github.com/jngz-es) | Amazon |
-| Kaituo Li | [kaituo](https://github.com/kaituo) | Amazon |
-| Lai Jiang | [wnbts](https://github.com/wnbts) | Amazon |
-| Sarat Vemulapalli | [saratvemulapalli](https://github.com/saratvemulapalli) | Amazon |
-| Tyler Ohlsen | [ohtyler](https://github.com/ohltyler) | Amazon |
-| Vamshi Vijay Nakkirtha | [vamshin](https://github.com/vamshin) | Amazon |
-| Vijayan Balasubramanian | [VijayanB](https://github.com/VijayanB) | Amazon |
-| Weicong Sun | [weicongs-amazon](https://github.com/weicongs-amazon) | Amazon |
-| Yaliang Wu | [ylwu-amzn](https://github.com/ylwu-amzn) | Amazon |
-| Yizhe Liu | [yizheliu-amazon](https://github.com/yizheliu-amazon) | Amazon |
\ No newline at end of file
+| Maintainer              | GitHub ID                                               | Affiliation |
+| ----------------------- | ------------------------------------------------------- | ----------- |
+| Hanguang Zhang          | [zhanghg08](https://github.com/zhanghg08)               | Amazon      |
+| Jack Mazanec            | [jmazanec15](https://github.com/jmazanec15)             | Amazon      |
+| Jing Zhang              | [jngz-es](https://github.com/jngz-es)                   | Amazon      |
+| Kaituo Li               | [kaituo](https://github.com/kaituo)                     | Amazon      |
+| Lai Jiang               | [wnbts](https://github.com/wnbts)                       | Amazon      |
+| Sarat Vemulapalli       | [saratvemulapalli](https://github.com/saratvemulapalli) | Amazon      |
+| Tyler Ohlsen            | [ohtyler](https://github.com/ohltyler)                  | Amazon      |
+| Vamshi Vijay Nakkirtha  | [vamshin](https://github.com/vamshin)                   | Amazon      |
+| Vijayan Balasubramanian | [VijayanB](https://github.com/VijayanB)                 | Amazon      |
+| Weicong Sun             | [weicongs-amazon](https://github.com/weicongs-amazon)   | Amazon      |
```

Comment:
Weicong already left.  Do we want to remove him?

Reply:
good point - updated